packages/use-contractkit/src/utils/useIsMounted.ts

```diff
@@ -0,0 +1,12 @@
+import { useRef, useEffect } from 'react';
+
+export function useIsMounted() {
```

Comment:
This isn't my favorite solution, but we had a lot of async state setting in callbacks that can cause memory leaks and it would be a lot of work to refactor those. This was an easy hook to add to track mounted status.